src/DTO/Channel.php

```diff
@@ -6,6 +6,22 @@
 
 use JsonSerializable;
 
+/**
+ * @psalm-type ptChannel = array{
+ *     active: bool,
+ *     client_id: int,
+ *     created: string,
+ *     description: string,
+ *     id: int,
+ *     meta: array,
```

Comment:
You need to define the meta.
In the Meta Class, create the Psalm data type for meta.
Then here, you need to define an array of that type.

Reply:
I forgot to mention, you can `import` the data type from the other class.
https://psalm.dev/docs/annotating_code/supported_annotations/#psalm-import-type